bin/incoming-mail-inject.pl

```diff
@@ -40,55 +41,46 @@ ()
         768 * 1024;
 }
 
-my $buf;
 my $msg = '';  # in-memory message
-my $rv;
-my $len = 0;
-my $ieh;
-my $ignore_message = 0;  # bool: to ignore rest of message.
+my $len = 0;   # length of message
+
 eval {
-    while ($rv = sysread(STDIN, $buf, 1024*64)) {
+    my ( $buf, $rv );
+    while ( $rv = sysread STDIN, $buf, 1024*64 ) {
         next if $ignore_message;
         $len += $rv;
-        if ($ieh) {
-            $ieh->append($buf);
-        } else {
-            $msg .= $buf;
-        }
-
-        if ($len > IN_MEMORY_THRES && ! $ieh) {
-            if (should_ignore($msg)) {
-                $ignore_message = 1;
-                next;
-            }
-
-            # allocate a mogile filehandle once we cross the line of
-            # what's too big to store in memory and in a schwartz arg
-            $ieh = LJ::IncomingEmailHandle->new;
```

Comment:
I think I was just grepping for mogclient and converting things and did IncomingEmailHandle before I got to the worker and realized I could just get rid of the handle. I'll remove the module.